Zend/tests/array_unpack/classes.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+Array unpacking with classes
+--FILE--
+<?php
+
+Class C {
+	public const ARR = [1, 2, 3];
+	public const FOO = [0, ...self::ARR, 4];

```

Comment:
Does this work if you swap the order of ARR and FOO? I think all of your current test cases here will get evaluated during compilation, but you'll see failures if the AST needs to be evaluated at runtime.